data/human/war reactions.txt

```diff
@@ -0,0 +1,257 @@
+			`	"Wait, you're a captain, right? Were you in the big fights? I was on Maker when I saw some Pug ships fighting human ships on our Syndicate school's big educational viewscreen."`
+			choice
+				`	"Yes, I'm Captain <first> <last>."`
+					goto me
+				`	"I blew up my share of Pug ships."`
+					goto blew
+				`	"Actually, I led the battle."`
+			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You're Captain <first> <last>! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
+				goto next
+			label me
+			`	"Oh, hold on," he says, pulling out a durable-looking data pad labeled 'Syndicate Childcare Device'. After some typing and swiping, he declares, "You led the attack! Wow! It's an honor to meet you, sir. Can I shake your hand?" He puts out his hand, and you shake it.`
+				goto next
+			label blew
+			`	"I bet you did! I wish I was on one of our warships, bravely defending the human race. The honor, the glory, the mercenary pay. It sounds amazing."`
+			label next
+			`	"What was it like, fighting the evil Pug?"`
+
+			choice
+				`	"Actually, they're not evil. Their 'war' was a ploy to get us to stop fighting among ourselves. A lot of Pug gave their lives willingly to save us from self-destruction."`
+				`	"War is hell, kid. I'm not going to glorify it for you."`
+					goto hell
+				`	"Leading a battle is like chess with guns and engines. It's the most exhilarating feeling I've ever felt."`
+					goto chess
+				`	"It was an honor to protect the human race from our attackers."`
+					goto honor
+			`	"That's so insightful! Still, without people like you, the Pug's ploy would never have worked. Thank you for your service, Captain."`
+				goto salute
+			label hell
+			`	"You went through hell for us, eh? That's something to think about. Maybe I should stick to engineering instead of becoming a combat pilot. I dunno."`
+				goto salute
+			label chess
+			`	"The Syndicate is training me to maintain and pilot combat ships! One day, I'll fight aliens like you did."`
+				goto salute
+			label honor
+			`	"Glory, honor, and service. The Syndicate has been training me, but I've often thought of enlisting in the Navy. It isn't as profitable, but it's certainly more noble."`
+			label salute
+			`	You arrive at the factory, and an older girl, perhaps seventeen, meets the two of you. Gus stands to attention and gives you a well-practiced salute. You politely salute him in return.`
+			`	As Gus ends his salute, the girl attacks him from behind, putting him into a choke hold. It's clear she's taking extreme care when doing it. "Did you wander out of the safe zone? Was it the spaceport landing area this time? I can see scratches and bruises all over you." Gus replies, "I'm sorry, Sally! I was curious."`
```

Comment:
If you change the specific nature of his workplace, I'd suggest aligning this to be something like "Did you try and take a shortcut again?" rather than the safe zone reference.

Reply:
He now works in the childcare district. Entering the nearby spaceport without supervision is exclusively a no-no for repair shop orphans.